spec/models/post_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'Post model' do
+    user = User.create(name: 'Ranjeet', bio: 'This is bio')
+    subject { Post.new(title: 'post title', text: 'post text', author_id: user) }
+    before { subject.save }
```

Comment:
- [ ]  Kindly add a test for the `update_posts_counter` method you have in your Post model as it is required to test all the custom methods in each model.